Documentation/contributing/development/images.rst

```diff
@@ -229,32 +221,8 @@ different value and then proceed with the steps below.
    to approve the build that was created by GitHub Actions `here <https://github.com/cilium/cilium/actions?query=workflow:%22Base+Image+Release+Build%22>`__.
    Note that at this step cilium-builder build failure is expected since we have yet to update the runtime digest.
 
-#. Wait for cilium-runtime build to complete. Only after the image is available run:
-
-   .. code-block:: shell-session
-
-       $ make -C images/ update-runtime-image update-builder-image
-
-#. Commit your changes and re-push to the PR in cilium/cilium.
-
-   .. code-block:: shell-session
-
-       $ git commit --amend -sa
-
-#. Ping one of the members of `team/build <https://github.com/orgs/cilium/teams/build/members>`__
-   to approve the build that was created by GitHub Actions `here <https://github.com/cilium/cilium/actions?query=workflow:%22Base+Image+Release+Build%22>`__.
-
-#. Wait for the build to complete. Only after the image is available run:
-
-   .. code-block:: shell-session
-
-       $ make -C images/ update-runtime-image update-builder-image
-
-#. Commit your changes and re-push to the PR in cilium/cilium.
-
-   .. code-block:: shell-session
-
-       $ git commit --amend -sa
+#. Wait for build to complete. The build will automatically generate one commit
+   with all the necessary changes across files in the repository.
```

Comment:
Where does the commit go, into the PR or into the master branch? Hence afterwards is it that you just close out the PR, or the PR needs to be merged before continuing to the next step?

Reply:
Changed to:
```
#. Wait for build to complete. The build will automatically generate one commit
   and push it to your branch with all the necessary changes across files in the
   repository. Once this is done the CI can be executed.
```